### ISSUE TITLE
Add voltage sensor

### ISSUE THREAD

**Issue by RikyUnreal** [mchwalisz/home-assistant-senec#46]
Hi all,
is it possible to add a sensor to also monitor the mains voltage? 

The id to grab from the web interface is "PM1OBJ1U_AC*"

![Voltage](https://github.com/mchwalisz/home-assistant-senec/assets/45981098/ac146f37-7ada-4aba-9605-8b8d57f31ff8)

**Comment by marq24** [comment]
if you like you can try my alternative fork...

**Comment by xpablo77** [comment]
> if you like you can try my alternative fork...

I would be interested too, what do you propose?

**Comment by RikyUnreal** [comment]
> if you like you can try my alternative fork...

I switched to your fork after the "https" update from Senec, and everything works like a charm (lots of new sensors, including voltage!).
Thanks a lot.

**Comment by klaus1956** [comment]
My question is: when using the updated branch do the entities keep the same or do I have to change all authorizations and dashboards?

**Comment by marq24** [comment]
@klaus1956 - to my "best" knowledge: When you keep the default device name [first entry field of the setup dialog] during the setup (which is 'senec') then the sensor-id's should be identical to mchwalisz integration version. (and based on the reply of RikyUnreal this statement seams to be valid)...

But I have to admit it's really a while ago, that I started with this fork and I am not 100% confident, that I have keept ALL the original "keys"... but it could be, that I just mix that up with a second integration that I also forked for my waterkotte heatpump... [there (waterkotte) I have changed some keys for sure]...

So when you give it a try (I know I repeat myself - but all you do you do on your own risk) - and it does not work... you simply can switch back to original version and you are fine...

**Comment by RikyUnreal** [comment]
> and based on the reply of RikyUnreal this statement seams to be valid

Yes, I confirm that it is sufficient to keep the same "device name".

The only issue is related to this: https://github.com/marq24/ha-senec-v3/issues/4
You need to add some Riemann Sum Integral sensors if you want to fully work with the Energy Dashboard.
